runtime/filetype.vim

```diff
@@ -1472,6 +1472,9 @@ au BufNewFile,BufRead *.markdown,*.mdown,*.mkd,*.mkdn,*.mdwn,*.md
 	\   setf markdown |
 	\ endif
 
+" Org (Emacs' org-mode)
+au BufNewFile,BufRead *.org			setf org
```

Comment:
can you please add a test to src/testdir/test_filetype.vim ?

Reply:
Done, I let me know if they are ok. I need to get better at doing tests :).

Reply:
Test is fine, actually we usually use a bit more simple tests, like this one: https://github.com/vim/vim/blob/4bd9b2b2467e696061104a029000e9824c6c609e/src/testdir/test_filetype.vim#L181